client/components/phone-input/index.jsx

```diff
@@ -16,21 +18,22 @@ import { countries } from './data';
 const PhoneInput = React.createClass( {
 	propTypes: {
 		onChange: React.PropTypes.func,
-		initialValue: React.PropTypes.string,
-		selectedCountryCode: React.PropTypes.string.isRequired,
+		value: React.PropTypes.string,
+		initialCountryCode: React.PropTypes.string,
 		countriesList: React.PropTypes.object.isRequired
 	},
 
 	getDefaultProps() {
 		return {
-			initialValue: '',
+			value: '',
 			onChange: noop,
+			initialCountryCode: 'us'
 		};
 	},
 
 	getInitialState() {
-		const selectedCountry = countries[ this.props.selectedCountryCode ];
-		const formattedNumber = formatNumber( this.props.initialValue, selectedCountry );
+		const selectedCountry = countries[ this.props.initialCountryCode ];
```

Comment:
Try to edit contact info for an already registered domain. Or update the contact info to the phone number I provided up there.

What I did is replaced the input field on the Edit Contact Info form with your `PhoneInput` and provided `countriesList` as it's already available there.